src/Screens/Home/MainPages/Wallet/index.tsx

```diff
@@ -1,17 +1,20 @@
 import React, { useState } from 'react';
-import { SafeAreaView } from 'react-native';
+import { View, SafeAreaView } from 'react-native';
 import { Text } from '@rneui/themed';
-import { CustomText, LoginBackground } from '../../../../components';
+import { walletStyles } from './styles';
+import RingGraph from '@components/RingGraph';
+
+
 
 const FriendsScreen = () => {
   const [accountBalance, setAccountBalance] = useState('0.00'); // need to fetch data from BE for this
-
   return (
-    <SafeAreaView
-      style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}
-    >
-      <Text style={{ fontSize: 50 }}>$ {accountBalance}</Text>
-      <Text>Cash balance</Text>
+    <SafeAreaView>
+        <RingGraph></RingGraph>
```

Comment:
You can rewrite this as `<RingGraph/>` if you're not going to pass props through it.  We might want to pass props through this component in the future though.